src/System.Data.SqlClient/src/PACKAGE.md

```diff
@@ -0,0 +1,54 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+Provides the .NET Data Provider for SQL Server. These classes provide access to versions of SQL Server and encapsulate database-specific protocols, including tabular data stream (TDS).
+
+System.Data.SqlClient is deprecated. Please use Microsoft.Data.SqlClient instead.
+
+## Key Features
+
+<!-- The key features of this package -->
+
+*
+
+## How to Use
+
+<!-- A compelling example on how to use this package with code, as well as any specific guidelines for when to use the package -->
+
+
+## Main Types
+
+<!-- The main types provided in this library -->
+
+The main types provided by this library are:
+
+- System.Data.SqlClient.SqlConnection
+- System.Data.SqlClient.SqlException
+- System.Data.SqlClient.SqlParameter
+- System.Data.SqlDbType
+- System.Data.SqlClient.SqlDataReader
+- System.Data.SqlClient.SqlCommand
+- System.Data.SqlClient.SqlTransaction
+- System.Data.SqlClient.SqlParameterCollection
+- System.Data.SqlClient.SqlClientFactory
+
+## Additional Documentation
+
+- [Introducing the new Microsoft.Data.SqlClient](https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/)
+- [https://techcommunity.microsoft.com/t5/sql-server-blog/announcement-system-data-sqlclient-package-is-now-deprecated/ba-p/4227205](Announcement: System.Data.SqlClient package is now deprecated)
+- For conceptual information about using this namespace when programming with .NET, see [SQL Server and ADO.NET](https://learn.microsoft.com/en-us/dotnet/framework/data/adonet/sql/).
+- API reference for System.Data.SqlClient can be found in: https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient?view=netframework-4.8.1
+- API reference for Microsoft.Data.SqlClient can be found in: https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient
+
+## Related Packages
+
+<!-- The related packages associated with this package -->
+
+- https://www.nuget.org/packages/Microsoft.Data.SqlClient
+
+## Feedback & Contributing
+
+<!-- How to provide feedback on this package and contribute to it -->
+
+System.Data.SqlClient is released as open source under the [MIT license](https://licenses.nuget.org/MIT). Bug reports and contributions are welcome at [the GitHub repository](https://github.com/dotnet/runtime).
```

Comment:
I think we want to discourage bug reports.
```suggestion
System.Data.SqlClient is released as open source under the [MIT license](https://licenses.nuget.org/MIT).
```

Reply:
Good point. I'll make the same change in all the PACKAGE.md files.